Client/mods/deathmatch/logic/luadefs/CLuaTimerDefs.cpp

```diff
@@ -111,6 +113,28 @@ int CLuaTimerDefs::KillTimer(lua_State* luaVM)
     return 1;
 }
 
+bool CLuaTimerDefs::IsTimerPaused(lua_State* luaVM, CLuaTimer* pLuaTimer)
+{
+    //  bool isTimerPaused ( timer theTimer )
+    CLuaMain* luaMain = m_pLuaManager->GetVirtualMachine(luaVM);
+    if (!luaMain)
+        return false;
+
+    bool bIsPaused = pLuaTimer->IsPaused();
+    return bIsPaused;
+}
+
+bool CLuaTimerDefs::SetTimerPaused(lua_State* luaVM, CLuaTimer* pLuaTimer, bool bPaused)
+{
+    //  bool setTimerPaused ( timer theTimer, bool paused )
+    CLuaMain* luaMain = m_pLuaManager->GetVirtualMachine(luaVM);
+    if (!luaMain)
+        return false;
+
+    luaMain->GetTimerManager()->SetTimerPaused(pLuaTimer, bPaused);
```

Comment:
Can you add a method `SetPaused` to the CLuaTimer class that will call `SetTimerPaused` on itself? Would be better to remove the lua_State parameter and just return `pLuaTimer->SetPaused(bPaused);`

Reply:
Eh, I think that would be a bit inconsistent considering the fact that all the other timer functions are primarily handled in `CLuaTimerManager`